python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -1472,6 +1471,9 @@ def submit(self, keep_going=None, keep_on_fail=None, prepare=False):
             # split into subjobs
             rjobs = self._doSplitting()
 
+            if not rjobs or len(rjobs) == 1 and rjobs[0] is self and self.splitter is not None:
```

Comment:
I mean a conditional like:
```
if self.splitter and len(self.subjobs) == 0:
    raise ...
```
If you have a splitter then subjobs should have been filled by this point. If not it's an error.